tasty-hunit-compat.cabal

```diff
@@ -4,7 +4,7 @@ cabal-version: 1.12
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: f6e3d0fadd3de7f69b21a4e939be2344dbe7a68595c7c47c845f3d2ef8fa155b
+-- hash: 57e2bf4d1e8058d57df15fe1e8a2c9c7f8d7e93985b9ff752ba852c809380b0c
```

Comment:
We should use a newer version as we did in `morley`, so that this hash is not generated.

Reply:
Do we really care here?

I mean, pros of this is that there will be less conflicts during rebases, which does not matter for such a small library. And cons are that we have to invoke `./scripts/generate-cabal-files.sh ` manually and witness the warning on each build

```
/home/martoon/serokell/tasty-hunit-compat/tasty-hunit-compat.cabal was modified manually. Ignoring /home/martoon/serokell/tasty-hunit-compat/package.yaml in favor of the cabal file.
If you want to use the package.yaml file instead of the cabal file,
then please delete the cabal file.
```

Reply:
> such a small library

Hm, nice reminder. With this in mind, I have an ultimate solution: not use `hpack` at all and have only `tasty-hunit-compat.cabal`. For single module library benefits of `hpack` sound quite negligible, but it adds some overhead.

Reply:
Well, I already slightly regret that I brought up this topic, but doing this actually makes sense.